pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -189,11 +189,8 @@ public static String convertBooleanActivityAnswerToString(Optional<Object> answe
 
     }
 
-    public boolean isParticipantConsented(@NonNull String ddpParticipantId, @NonNull DDPInstanceDto ddpInstanceDto) {
-        ElasticSearchParticipantDto participant = ElasticSearchUtil.getParticipantESDataByParticipantId(
-                ddpInstanceDto.getEsParticipantIndex(), ddpParticipantId);
-        return (participant != null && hasParticipantConsentedToSharedLearning(participant, ddpInstanceDto))
-                && hasParticipantConsentedToTumor(participant);
+    public boolean isParticipantConsented(@NonNull ElasticSearchParticipantDto participant, @NonNull DDPInstanceDto ddpInstanceDto) {
```

Comment:
Instead of passing in an id and querying ES, we're now passing in a fully-baked Dto so that the caller in the tests can change the data more easily.